.github/workflows/e2e-ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node: [12, 14, 16]
+        node: [14, 16, 18]
```

Comment:
At first, I was "wait, we still support node 12!", but we are dropping its support in 13 days, so yeah, let's do this 🥳